BeardedSpice/BSVolumeControlProtocol.h

```diff
@@ -0,0 +1,30 @@
+//
+//  BSVolumeControlProtocol.h
+//  BeardedSpice
+//
+//  Created by Roman Sokolov on 15.05.17.
+//  Copyright (c) 2015-2017 GPL v3 http://www.gnu.org/licenses/gpl.html
+//
+
+#ifndef BSVolumeControlProtocol_h
+#define BSVolumeControlProtocol_h
+
+typedef enum {
```

Comment:
About "BSVolumeControl-Protocol.h" - Not important, maybe next time.
NS_ENUM - This is interesting :)